src/components/sidebar/nodeView/NodeView.tsx

```diff
@@ -43,11 +43,11 @@ class NodeView extends React.Component
         };
     }
 
-    public componentDidMount() {
+    public async componentDidMount() {
         const selectedNodeId = this.props.match!.params.id;
         if (selectedNodeId) {
             this.props.mapStore!.setSelectedNodeId(selectedNodeId);
-            this.queryNode(selectedNodeId);
+            await this.queryNode(selectedNodeId);
```

Comment:
Just out of interest, did the old code introduce problems? Or was this just modified since it is nicer? In my opinion it is a good change, so I don't question that. I just wonder if fireAndForget style executions in lifecycle methods introduce problems